docs/ashell.md

```diff
@@ -86,6 +86,7 @@ Commands list:
      run [FILE] Runs the JavaScript program in the file
    parse [FILE] Check if the JS syntax is correct
     stop Stops current JavaScript execution
+ bootcfg [FILE] Set the file that should run at boot
```

Comment:
*just thinking* -- `bootcfg` needs a better name, we don't have autocomplete. Even `boot` would suffice since the text tells it's run at next boot. Or `default` might also work (default script is the one that is run at next boot).